components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -690,6 +695,26 @@ void RewardsServiceImpl::LoadPublisherInfoList(
     uint32_t limit,
     ledger::PublisherInfoFilter filter,
     ledger::GetPublisherInfoListCallback callback) {
+  auto now = base::Time::Now();
```

Comment:
why do we have two separate methods for this? What if the code calling this method already set month and year to something else? They are going to get unexpected results. If you are going to predefine filter values then you shouldn't have the filter as a parameter

Reply:
fixed in https://github.com/brave/brave-core/pull/1029